src/components/toast/toast.stories.js

```diff
@@ -55,6 +55,32 @@ Basic.story = {
   },
 };
 
+export const CustomWidth = () => {
```

Comment:
I don't think we need to create a story here - let's make it in the `mdx`

Reply:
I agree with @DlgSHi the `MDX` is enough for this example